commit ce30bd427fa0c49d8264d14f1d7a7bf7b7d21a73
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Fri Mar 8 10:49:17 2019 +0000

    staging: vc-sm-cma: Remove the debugfs directory on remove
    
    Without removing that, reloading the driver fails.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c b/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
index 4bc738a2abf3..f70e3e7aedb3 100644
--- a/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
+++ b/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
@@ -757,7 +757,7 @@ static int bcm2835_vc_sm_cma_remove(struct platform_device *pdev)
 	pr_debug("[%s]: start\n", __func__);
 	if (sm_inited) {
 		/* Remove all proc entries. */
-		//debugfs_remove_recursive(sm_state->dir_root);
+		debugfs_remove_recursive(sm_state->dir_root);
 
 		/* Stop the videocore shared memory service. */
 		vc_sm_cma_vchi_stop(&sm_state->sm_handle);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1377_ce30bd427fa0c49d8264d14f1d7a7bf7b7d21a73.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1377_ce30bd427fa0c49d8264d14f1d7a7bf7b7d21a73.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
